### PR TITLE
Create index if necessary

### PR DESCRIPTION
**Pull request by jsf9k**
Also some minor updates made while updating the code to help restore the DMARC Elasticsearch database.

### CODE REVIEW

File: copy_old_es_database.py [hunk 1]
```diff
@@ -52,12 +141,37 @@ def main():
...
+            response = requests.put(index_only_url,
+                                    auth=new_awsauth,
+                                    json=INDEX_PAYLOAD,
+                                    headers={'Content-Type': 'application/json'},
+                                    timeout=300)
+            # Raises an exception if we didn't get back a
+            # 200 code
+            response.raise_for_status()
+        except requests.exceptions.RequestException:
+            logging.exception("Unable to create the index 'report.")
```
Comment (dav3r) [inline copy_old_es_database.py]: Shouldn't this be:
```
logging.exception("Unable to create the index 'dmarc_aggregate_reports'.")
```
Or am I misunderstanding something?

**Review by dav3r — APPROVED** [review]
Looks good, except for one minor logging question.

File: copy_old_es_database.py (hunk 1 above)
Comment (jsf9k) [reply copy_old_es_database.py]: Good eye.  Fixed in 0c4fa15.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✅

**Pull request merged**